### PR TITLE
Remove all ampersands from the service

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
It does what it says on the tin!

(Dan says we should keep it for the department name)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ gem "bootsnap", ">= 1.4.4", require: false
 gem "govuk-components"
 # GOV UK component form builder DSL
 gem "govuk_design_system_formbuilder"
-# Turbo & Stimulus
+# Turbo and Stimulus
```
Comment (baarkerlounger) [inline Gemfile:22]: :joy: Thorough!

Comment (paulrobertlloyd) [reply Gemfile:22]: Always.

File: config/initializers/active_admin.rb [hunk 2]
```diff
@@ -201,7 +201,7 @@
   #
   # config.create_another = true
 
-  # == Register Stylesheets & Javascripts
```
Comment (baarkerlounger) [inline config/initializers/active_admin.rb]: I would revert the ones in this file so as not to cause unnecessary conflicts when upgrading versions.

Comment (paulrobertlloyd) [reply config/initializers/active_admin.rb]: Good idea! Done.

**Review by baarkerlounger — APPROVED** [review]

**Pull request merged**